### ISSUE TITLE
UnusedUses: false positives with groups in tests

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#469]
Code to reproduce:

```php
use Foo\Bar\Something;

class MyTestCase {

    /**
     * @group something // Case of reference name "something" and use statement "Something" do not match.
     */
    public function testShouldReturnTrue(): void
    {
    }
}
```

**Comment by carusogabriel** [comment]
Also, https://grammarly.com reports a typo in the message: should use `does` instead of `do`.

**Comment by kukulich** [comment]
See options of this sniff: https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesunuseduses-
